app/src/main/java/com/example/playlistmaker/player/ui/view_model/FragmentPlayerViewModel.kt

```diff
@@ -5,17 +5,23 @@ import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
+import com.example.playlistmaker.R
+import com.example.playlistmaker.media.data.Playlist
+import com.example.playlistmaker.media.data.PlaylistsState
+import com.example.playlistmaker.media.data.entity.TracksInPlaylistEntity
 import com.example.playlistmaker.media.domain.db.LikedTracksInteractor
+import com.example.playlistmaker.media.domain.db.PlaylistsInteractor
 import com.example.playlistmaker.player.domain.PlayerInteractor
 import com.example.playlistmaker.player.domain.Track
 import com.example.playlistmaker.player.ui.PlayerState
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 
-class PlayerViewModel(
+class FragmentPlayerViewModel(
```

Comment:
🍏 Рекомендую не указывать во вьюмодели тип реализации вью (активити, фрагмент, просто вью)
```suggestion
class PlayerViewModel(
```